boards/raytac/an54l15q_db/raytac_an54l15q_db_nrf54l15_cpuapp_ns.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2025 Nordic Semiconductor ASA
+# Copyright (c) 2025 Raytac Corporation.
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+identifier: raytac_an54l15q_db/nrf54l15/cpuapp/ns
+name: Raytac-AN54L15Q-DB-nRF54l15-Application-Non-Secure
+type: mcu
+arch: arm
+toolchain:
+  - gnuarmemb
+  - zephyr
+ram: 256
+flash: 1524
```

Comment:
Shouldn't these values be rather 80 for RAM and 844 for flash?

Reply:
I checked Nordic's nRF54L15-DK, and the settings are the same.